webpack-demo/src/modules/clear.js

```diff
@@ -0,0 +1,10 @@
+/*eslint-disable*/
```

Comment:
You are disabling linters for each js files of this project. Please don't disable linters.
There are a few reasons for using linters:
Catching syntax errors is more efficient. There is no need to debug simple mistakes like typos - the linter does it for you.
The entire codebase looks like written by one person.
Programmers can focus on solving problems, instead of cleaning up the code.

You can search for the linter error on their [documentation page](https://eslint.org/docs/latest/) and solve it.